sys-apps/tuned/tuned-2.13.0.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit python-single-r1 systemd xdg-utils
+
+DESCRIPTION="Daemon for monitoring and adaptive tuning of system devices"
+HOMEPAGE="https://fedorahosted.org/tuned/"
+SRC_URI="https://github.com/redhat-performance/tuned/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+CDEPEND="
```

Comment:
rename to `COMMON_DEPEND`

Reply:
Also due to copying. Fixing.

Reply:
however, `DEPEND` contains none else than `COMMON_DEPEND` so this var is just superfluous. Just make it `DEPEND` here already and inherit that in `RDEPEND`.